crates/forge/Cargo.toml

```diff
@@ -46,6 +46,7 @@ scarb-metadata = { git = "https://github.com/software-mansion/scarb.git", rev =
 project-root = "0.2.2"
 console = "0.15.7"
 snapbox = "0.4.11"
+starknet = "0.5.0"
```

Comment:
Please revert this change + changes made to `Cargo.lock`

Reply:
I assumed the cheactcode would follow the same style as io. I will fix up.